<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VLayout_1">
    <item>
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>Input Shapefile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="inShape"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VLayout_2">
    <item>
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>Target field</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="inField"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VLayout_5">
    <item>
     <widget class="QLabel" name="label5">
      <property name="text">
       <string>Spatial Autocorrelation Result</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="SAresult"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VLayout_3">
    <item>
     <widget class="QLabel" name="label3">
      <property name="text">
       <string>Weights FIle ID Variable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="idVariable"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Contiguity Weight</string>
   </property>
   <widget class="QRadioButton" name="queen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>359</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Queen Contiguity</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rook">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>359</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rook Contiguity</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
